specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2022-02-14/imagebuilder.json

```diff
@@ -1226,6 +1226,7 @@
               "items": {
                 "$ref": "#/definitions/ImageTemplateInVMValidator"
               },
+              "x-ms-identifiers": [],
```

Comment:
From documentation (http://azure.github.io/autorest/extensions/#x-ms-identifiers), this extension seems to be "InProgress" and not ready. So why are we adding this?

Reply:
i added this as it was causing the lintdiff check to fail. perhaps @honghr can help answer this question.

Reply:
Hi @sumit-kalra @zdelacerda-microsoft The extension is ready now, but the doc may have some confusion. We will ask the author to fix it. Thanks.